### PR TITLE
Created testing for each API endpoint

### PR DESCRIPTION
**Pull request by donnatopia**
1. I used a package called postman-request and jest to create a basic test for each of my API endpoints
2. I have an example folder within my server that holds the data which I test against.

This pull request also has my commits from my other pull request for the complex queries

### CODE REVIEW

**Comment by donnatopia** [comment]
I created a file to check for query times within my db folder and reorganized my folders

File: example.env [hunk 1]
```diff
@@ -6,5 +6,6 @@
 # SERVER ENV
 PORT=''
 USER=''
+LOCALHOST=''
```
Comment (wooseok0717) [inline example.env:9]: do you still need this file?

File: package.json [hunk 2]
```diff
@@ -31,11 +32,13 @@
     "mongoose": "^6.9.1",
     "morgan": "^1.10.0",
     "pg": "^8.9.0",
+    "postman-request": "^2.88.1-postman.32",
     "stream": "^0.0.2"
   },
   "devDependencies": {
     "eslint": "^8.34.0",
     "eslint-config-airbnb": "^19.0.4",
+    "jest": "^29.4.3",
```
Comment (wooseok0717) [inline package.json:41]: do you suppose use postman-request library with jest?

File: server/MVC/controllers.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        res
+          .status(200)
+          .json(result);
+      })
+      .catch((err) => {
+        console.log(err);
+        res.sendStatus(404);
+      })
+  }
+}
```
Comment (wooseok0717) [inline server/MVC/controllers.js:42]: love your clean code ❤️

File: server/MVC/models.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      AS data
+      FROM products
+      WHERE id = $1`,
+      values: [id]
+    }
+
+    return await client
+      .query(product)
+      .then(({ rows }) => rows[0].data);
+  },
```
Comment (wooseok0717) [inline server/MVC/models.js:33]: not in a stage to review this part yet but I will try to come back to it asap :)

**Review by wooseok0717 — APPROVED** [review]
Awesome Job! makes me feel more behind but thank you for clean and readable code! I enjoyed doing code review!

File: example.env (hunk 1 above)
Comment (donnatopia) [reply example.env:9]: I created the example.env file for other people to use if they chose to clone my repo!

File: package.json (hunk 2 above)
Comment (donnatopia) [reply package.json:41]: yepp i used postman-request with jest

File: server/MVC/controllers.js (hunk 3 above)
Comment (donnatopia) [reply server/MVC/controllers.js:42]: thank you!!

File: package.json (hunk 2 above)
Comment (wooseok0717) [reply package.json:41]: nice awesome!

**Pull request merged**